cmd/api/main.go

```diff
@@ -65,7 +68,7 @@ func main() {
 	serverConfig := config.Get()
 
 	log := logger.Get(serverConfig.LOG_LEVEL)
-	// This defer statement will be executed at the end of the application lifecycle, so we do not lose
+	// This defers statement will be executed at the end of the application lifecycle, so we do not lose
```

Comment:
Not sure if I missed this in my previous review, but this looks like auto correction giving bad advise. Can you fix it quick please. The rest is LGTM.

Reply:
I mean revert that change.

Reply:
Ohh!! Auto correct, didn't notice. 
Reverted the change.